### PR TITLE
Update materialize framework

### PR DESCRIPTION
**Pull request by impmihai**
The materialize framework was left behind with the
updates. Used version v0.97.8, updated version v0.98.0

### CODE REVIEW

**Comment by Mariatta** [comment]
ack 47163dc

File: vendors/materialize/css/materialize.css [hunk 1]
```diff
@@ -1020,11 +1020,11 @@
 }
 
 .teal.lighten-1 {
-  background-color: #424242 !important;
+  background-color: #26a69a !important;
```
Comment (Mariatta) [inline vendors/materialize/css/materialize.css]: Actually... what's with the all color changes?
Will this affect anything in our current landing page?

Comment (impmihai) [inline vendors/materialize/css/materialize.css]: I double checked everything in the pages and they look the same. I just saw that too. This is really strange.

**Comment by Mariatta** [comment]
unack 47163dc

**Comment by hemangsk** [comment]
ack 3cc6def

**Comment by hemangsk** [comment]
@impmihai this is perfect, are you planning to do the autocomplete chip stuff in this PR itself?

**Comment by sils** [comment]
you didn't answer - what are all the colour changes for?

**Comment by sils** [comment]
(or at least I didn't ge tit)

**Comment by hemangsk** [comment]
@sils There are these two color changes  # 212121 (materialize default light red-ish) to # 424242 (greyish) and  # ea4a4f to # ee6e73, overriding materialize's default colors

**Comment by impmihai** [comment]
@hemangsk how do you suggest? I can do both . I am already working on another branch, but I can copy those files here too I guess.

**Pull request merged**